eventlog/signals/active_users_health_check.py

```diff
@@ -25,10 +27,46 @@ def active_users_health_check(sender, health_request: HealthCheckRequest, **kwar
         emojis = [":nerd_face:", ":thinking_face:", ":face_with_monocle:", ":face_with_cowboy_hat:"]
         emoji = emojis[randint(0, len(emojis) - 1)]
 
+    user_previews = [UserPreview(user).preview for user in active_users_queryset]
+
     return HealthCheckRecentActivity(
         emoji=emoji,
         name="Active Users",
         amount=count,
         extra=", ".join(list(active_users)),
+        preview=user_previews,
         stand_alone=True  # always give active users its own line
     )
+
+
+@receiver(signal=health_check_signal)
+def email_health_check(sender, health_request: HealthCheckRequest, **kwargs):
+    from email_manager.models import EmailLog
+    recent_emails = EmailLog.objects.filter(created__gte=health_request.since, created__lt=health_request.now)
+    count = recent_emails.count()
+    if count:
+        return HealthCheckRecentActivity(
+            emoji=":email:",
+            name="Emails Sent",
+            amount=count,
+            extra=", ".join([email.subject for email in recent_emails]),
+            stand_alone=True,
+            preview=[email.preview for email in recent_emails]
+        )
+
+
+@receiver(signal=health_check_signal)
+def discordance_triage_health_check(sender, health_request: HealthCheckRequest, **kwargs):
+    recent_reports = DiscordanceReportTriage.objects.filter(
+        modified__gte=health_request.since,
+        modified__lt=health_request.now)
+    count = recent_reports.count()
+    if count:
```

Comment:
super minor thing, but from Python 3.8 onwards you can do

```
if count := recent_reports.count():
    return HealhCheckRecentActivity(
```

Where the nested block is only executed if count has a true value (e.g. a number other than 0)